doc/user_guide/dynamo0p3.rst

```diff
@@ -423,6 +437,20 @@ associated kernel metadata description and their precision:
 +--------------------------+---------------------------------+-----------+
 | LOGICAL(L_DEF)           | GH_SCALAR, GH_LOGICAL           | L_DEF     |
 +--------------------------+---------------------------------+-----------+
+| REAL(R_DEF)              | GH_ARRAY, GH_REAL               | R_DEF     |
```

Comment:
I think these new lines can be merged with the corresponding lines for scalars by altering the latter to say "GH_SCALAR/GH_ARRAY, GH_INTEGER" etc. In fact, once we've discussed my suggestion of `GH_SCALAR*n` these changes might not be required at all.